setupPRarea

```diff
@@ -0,0 +1,24 @@
+#!/bin/tcsh
+
+setenv SCRAM_ARCH slc6_amd64_gcc493
+setenv PATH /afs/cern.ch/cms/common/:$PATH
```

Comment:
don't use full paths.
do not hardcode SCRAM_ARCH.
The starting point should be that user already has knowledge of at least one release and scram is in the path.

These can be left commented out as suggestions.

BTW, I have no intention to support tcsh scripts.